src/test/java/uk/ac/ebi/eva/commons/models/converters/data/MongoDBObjectToVariantConverterTest.java

```diff
@@ -25,13 +24,13 @@
 public class MongoDBObjectToVariantConverterTest {
```

Comment:
The description was OK! :) 
"Heavily based upon DBObjectToVariantConverterTest"